vmsize_test.go

```diff
@@ -0,0 +1,192 @@
+package skewer
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type VMSizeType struct {
+	family           string
+	subfamily        *string
+	cpus             string
+	cpusConstrained  *string
+	additiveFeatures []rune
+	acceleratorType  *string
+	version          string
+	series           string
+}
```

Comment:
Should this only be used in the test type? Why are we doing the map structure instead of a strongly typed dsa?